shell/pages/c/_cluster/apps/charts/install.vue

```diff
@@ -758,7 +758,10 @@ export default {
 
       // Load a values component for the UI if it is named in the Helm chart.
       if ( component ) {
-        if ( this.$store.getters['catalog/haveComponent'](component) ) {
+        const hasChartComponent = this.$store.getters['type-map/hasCustomChart'](component);
+
+        if ( hasChartComponent ) {
+          this.valuesComponent = this.$store.getters['type-map/importChart'](component);
```

Comment:
Don't think we need both lines that set `this.valuesComponent` - assume its only the first one?

Reply:
Ah good catch, left that in there by mistake. Removed `'catalog/importComponent'`